single-node/calico.yaml

```diff
@@ -0,0 +1,185 @@
+# This ConfigMap is used to configure a self-hosted Calico installation.
```

Comment:
Pointed the single node to the same file and removed the extra, spun up the single node to make sure it went ok, looks good.